### PR TITLE
build update

### PR DESCRIPTION
**Pull request by zaqqwerty**
removed some unused packages

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -97,33 +97,3 @@ tf_workspace(tf_repo_name = "@org_tensorflow")
...
-  visibility = ["//visibility:public"],
-)
-    """,
-    sha256 = "7e7a57e33c59280a17a66e521396cd8b1a55d0676c9f807078522fda52114b5c",
-    strip_prefix = "eigen-eigen-8071cda5714d",
-    urls = [
-        "https://storage.googleapis.com/mirror.tensorflow.org/bitbucket.org/eigen/eigen/get/8071cda5714d.tar.gz",
-        "https://bitbucket.org/eigen/eigen/get/8071cda5714d.tar.gz",
-    ],
-)
```
Comment (jaeyoo) [inline WORKSPACE:117]: Wait, we didn't remove the `qsim_old` at all, so still there is a dependency on the `Eigen` in [here](https://github.com/tensorflow/quantum/blob/0c90033d8763478300a6626b038c7c7ddacb7a4a/tensorflow_quantum/core/qsim/unitary_space_test.cc#L18-L21). Shall we remove this BUILD lines after removing the entire dependent codes?

File: tensorflow_quantum/core/qsim/BUILD [hunk 2]
```diff
@@ -174,7 +174,6 @@ cc_test(
         ":unitary_space",
         "//tensorflow_quantum/core/src:circuit",
         "@com_google_googletest//:gtest_main",
-        "@eigen//:eigen3",
```
Comment (jaeyoo) [inline tensorflow_quantum/core/qsim/BUILD:177]: Is it really ok even if the `qsim_old` code requires `Eigen::**`?

**Comment by zaqqwerty** [comment]
Good catch, seems we should not remove these until we remove `qsim_old`.  @MichaelBroughton are we good to remove the `qsim_old` yet?  I saw Unitary Feature got merged into external QSim, wasn't sure which pieces were still moving there

**Comment by MichaelBroughton** [comment]
We’re getting close on external qsim. They need a few more tweaks and then to cut a release. From there we update our unitary op. Then we can cut qsim_old :).

**Comment by zaqqwerty** [comment]
Will close pending `qsim_old` removal

**Pull request closed without merging**